package.json

```diff
@@ -45,6 +45,18 @@
         "roby"
     ],
     "contributes": {
+        "configuration": {
+            "type": "object",
+            "title": "Rock Workspace Configuration",
+            "properties": {
+                "rock.devFolder": {
```

Comment:
I don't understand what is this for

Reply:
It's just a simple way to be already in your main dev folder when doing "Add Workspace".

Reply:
Ah, cool..